drivers/interrupt_controller/intc_gd32_exti.c

```diff
@@ -82,7 +82,7 @@ static const uint8_t line2irq[NUM_EXTI_LINES] = {
 #endif /* CONFIG_SOC_SERIES_GD32F4XX */
 };
 
-__unused static void gd32_exti_isr(void *isr_data)
+__unused static void gd32_exti_isr(const void *isr_data)
```

Comment:
Was this intended?

Reply:
the script works by steps, and it first checks about const qualifier on isr used through IRQ_CONNECT, so yes it's fine.